src/containers/ProgressNavBar.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <header
+    class="relative flex h-16 w-full flex-col items-center justify-between"
+  >
+    <div
+      class="flex h-16 w-full items-center justify-between border-gray_02-light p-4 dark:border-gray_02-dark"
+    >
+      <BackButton :prevPage="props.prevPage" />
+      <CloseButton />
+    </div>
+    <div
+      class="absolute bottom-0 h-[2px] bg-gray_05-light dark:bg-gray_05-dark left-0"
+      :class="processWidth[props.pageName]"
```

Comment:
동적 함수로 get~Style() 생성하여 하나로 단축해주세요.


```suggestion
      :class={`absolute bottom-0 h-[2px] bg-gray_05-light dark:bg-gray_05-dark left-0 ${get~Style()}`}
```

```
const get~Style = () => {
  return {
  'w-1/3' : props.pageName === 'size'
  'w-2/3' : props.pageName === 'ingredient' 
  'w-full' : props.pageName === 'result'
}
}
```